### PR TITLE
Create static pages for application

### PR DESCRIPTION
**Pull request by ducminhvu**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title = "")
```
Comment (phamkykhoi) [inline app/helpers/application_helper.rb]: Remove ( )

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+    base_title = "Framgia e-learning"
```
Comment (phamkykhoi) [inline app/helpers/application_helper.rb]: use I18N

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
+    if page_title.empty?
+      base_title;
```
Comment (phamkykhoi) [inline app/helpers/application_helper.rb]: Remove ;

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,10 @@
...
+    else
+      page_title + " | " + base_title;
```
Comment (phamkykhoi) [inline app/helpers/application_helper.rb]: Remove ;

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About", about_path %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: Use I18n

File: app/views/layouts/_header.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "Framgia e-learning", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", root_path %></li>
+        <li><%= link_to "Help", help_path %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Use I18n

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+        <li><%= link_to "Log in", "#" %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Use I18n

File: app/views/layouts/_footer.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+      <li><%= link_to "Contact", contact_path %></li>
+      <li><a href="http://news.railstutorial.org/">News</a></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_footer.html.erb]: use link_to

File: app/views/layouts/_footer.html.erb [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+    </ul>
+  </nav>
+</footer>
```
Comment (vjnhat) [inline app/views/layouts/_footer.html.erb]: new line

File: app/views/layouts/_shim.html.erb [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```
Comment (vjnhat) [inline app/views/layouts/_shim.html.erb]: new line

File: app/views/static_pages/about.html.erb [hunk 11]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, "About") %>
```
Comment (phamkykhoi) [inline app/views/static_pages/about.html.erb]: Use I18n

File: app/views/static_pages/home.html.erb [hunk 12]
```diff
@@ -0,0 +1,8 @@
+<% provide(:title, "Home") %>
```
Comment (phamkykhoi) [inline app/views/static_pages/home.html.erb]: Use I18n

File: app/views/static_pages/about.html.erb [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+
+<div class="jumbotron">
+  <h1>StaticPages#About</h1>
+  <p>This is about web page</p>
+</div>
```
Comment (vjnhat) [inline app/views/static_pages/about.html.erb]: new line

File: app/views/static_pages/home.html.erb [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
+
+<div class="jumbotron">
+  <h1>Welcome to Framgia e-learning</h1>
+  <h2>This is the home page for Framgia e-learning center.</h2>
+
+  <%= link_to "Sign up now!", "#", class: "btn btn-lg btn-primary" %>
+</div>
```
Comment (vjnhat) [inline app/views/static_pages/home.html.erb]: new line

File: app/views/static_pages/home.html.erb [hunk 15]
```diff
@@ -0,0 +1,8 @@
+<% provide(:title, "Home") %>
+
+<div class="jumbotron">
+  <h1>Welcome to Framgia e-learning</h1>
+  <h2>This is the home page for Framgia e-learning center.</h2>
```
Comment (vjnhat) [inline app/views/static_pages/home.html.erb]: i18n

File: app/views/static_pages/contact.html.erb [hunk 16]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, "Contact") %>
+
+<div class="jumbotron">
+  <h1>StaticPages#contact</h1>
+  <p>Find me in app/views/static_pages/help.html.erb</p>
+</div>
```
Comment (vjnhat) [inline app/views/static_pages/contact.html.erb]: new line

File: app/assets/stylesheets/static_pages.scss [hunk 17]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (vjnhat) [inline app/assets/stylesheets/static_pages.scss]: if you not use this file, use git rm to remove it

File: app/views/static_pages/home.html.erb [hunk 18]
```diff
@@ -0,0 +1,8 @@
+<% provide(:title, t("home")) %>
+
+<div class="jumbotron">
+  <h1>Welcome to Framgia e-learning</h1>
+  <h2>This is the home page for Framgia e-learning center.</h2>
+
+  <%= link_to "Sign up now!", "#", class: "btn btn-lg btn-primary" %>
```
Comment (vjnhat) [inline app/views/static_pages/home.html.erb]: i18n?

**Pull request closed without merging**